src/hypothesis/stateful.py

```diff
@@ -145,6 +145,18 @@ def execute_step(self, step):
         """Execute a step that has been previously drawn from self.steps()"""
         raise NotImplementedError(u'%r.execute_step()' % (self,))
 
+    def print_start(self):
```

Comment:
With a nod to #1191, I think this new method should be documented :stuck_out_tongue:

Reply:
You're right, it should, although it's not exactly public API at the moment (maybe it should be, but right now I doubt that *anyone* is writing their own `GenericStateMachine` subclasses).

Reply:
Wait, it is. All of these methods already have docstrings.

They're not in the documentation itself, but that's I think OK for now - they're currently effectively an internal only API.

Reply:
I'm happy to leave this to the autoclass docs envisaged by #1190.